ci-operator/config/openshift/build-test-images/openshift-build-test-images-master.yaml

```diff
@@ -0,0 +1,35 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-openshift-4.6
+images:
+- context_dir: roots2i
+  dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    src:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/build-test-images/roots2i/.
+  to: test-build-roots2i
+promotion:
+  name: "4.7"
+  namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
```

Comment:
strictly speaking you do not need this tag_specification section, you do not need to construct a release payload because you're not doing any testing against PRs to this repo.

Reply:
ok I'll delete it and give the rehearsals a go

Reply:
yeah the CI rehearse/image jobs are not happy after removing tag_specification ... see below

Reply:
I've dropped the commit that removed it

Reply:
@bparees per https://github.com/openshift/release/blob/master/ci-operator/config/openshift/sippy/openshift-sippy-master.yaml should I get rid of both `promotion` and `tag_specification` then? ... since just removing `tag_specification` produced errors

Reply:
imho this is a bug in ci-operator.  You should not need to have a `tag_specification` specified to be able to use the `promotion` directive.  

tag_specification says "please build a release payload from this imagestream and add my newly built image content to it"

promotion says "please push these images i just built up to this imagestream"

@stevekuznetsov ?

But yes, given the behavior it looks like you need a tag_specification for now to make ci-operator happy, since you are using promotion.

Reply:
link to failed rehearsal for posterity: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/14178/rehearse-14178-pull-ci-openshift-build-test-images-master-images/1337506336295882752

Reply:
(and you need the promotion directive, that's ultimately how you're going to make this image available to your tests, or at least to the thing that mirrors it to quay)

Reply:
10-4 @bparees ... leaving both